packages/jest-config/src/normalize.ts

```diff
@@ -1175,6 +1178,21 @@ export default async function normalize(
     newOptions.shard = parseShardPair(argv.shard);
   }
 
+  // Process plugins
+  if (newOptions.plugins && newOptions.plugins.length > 0) {
```

Comment:
```suggestion
  if (newOptions.plugins.length > 0) {
```
would this work? Is `newOptions.plugins` always not `null`?